src/System.Management.Automation/engine/CommandParameter.cs

```diff
@@ -15,12 +15,14 @@ internal sealed class CommandParameterInternal
     {
         private class Parameter
         {
+            internal Ast ast;
             internal IScriptExtent extent;
```

Comment:
This API is used in many places where the AST is not available e.g. the PowerShell API, F&O. In those cases dummy extents are explicitly passed in. So both the Ast and extent members are required.